ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
+
+# Please note, this is an early version. All the verbose-stuff will be reduced at later stages.
+# TODO: functions for copy finished data to web folder
+# TODO: Information to slims?
+# TODO: What to do with original folders?
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
+pythonpath = "/apps/bio/software/anaconda2/envs/hcp/bin/python3"  # since server (as for now) has python2 installed
+
+parser = argparse.ArgumentParser(description='Wrapper to find non-processed folders, run ALU_parser on them.')
+parser.add_argument("-p", "--path", help="Path to folder with sequence-subfolders.", action="store", default=".")
+# /seqstore/remote/share/crc
+parser.add_argument("-r", "--run", help="Run for real. If not specified, just dry-run without any changes.",
+                    action="store_true")
+parser.add_argument("-v", "--verbose", help="Show more information while running.", action="store_true")
+args = parser.parse_args()
+sendmail = False
+seqpath = args.path
+
+
+def email_general(msg_to, msg_from, subject, body, attachment=None):
+    msg = EmailMessage()
+    msg.set_content(f'{body}\n'
+                    f'\n'
+                    f'Kind regards,\n'
+                    f'Clinical Genomics Gothenburg')
+    msg['Subject'] = f'{subject}'
+    msg['From'] = msg_from
+    msg['To'] = msg_to
+    # msg['Cc'] = "clinicalgenomics@gu.se"
+    if attachment:  # Add attachment if provided
+        csv_filename = os.path.basename(attachment)
+        with open(attachment, 'rb') as f:
+            data = f.read()
+            msg.add_attachment(data, maintype='text', subtype='plain', filename=csv_filename)
+    s = smtplib.SMTP('smtp.gu.se')
+    s.send_message(msg)
+    s.quit()
+
+
+def vcfparse(updated_file, v):
+    # TODO: Rev this part / rewrite
+    new_vcf = open(updated_file, 'w')
+    with open(v, "r") as vcf:
+        info_list = []
+        for line in vcf:
+            if ("ALU" in line or "LINE1" in line) and not line.startswith('#'):
+                info = line.split(';')
+                matching = [s for s in info if "SVLEN" in s]
+                match = str(matching).split('=')
+                length = match[1]
+                pos = (line.split('\t'))[1]
+                end = int(pos) + int(length.strip("']"))
+                info.insert(1, f"END={end}")
+                info_list.append(';'.join(info))
+            else:
+                info_list.append(line)
+    new_vcf.write(''.join(str(e) for e in info_list))
+    new_vcf.close()
```

Comment:
There is a nice package called cyvcf2 for parsing VCF files. No need for making our own parser.
https://www.genomoncology.com/blog/parsing-variants-using-the-cyvcf2-library

Reply:
Using a parser for this minor change seems like shooting rabbits with an elephant gun. We just want to add a small part with the total lenght to a simple text-file.
So, either we do this in a simple function like this, or we install third-party packages to read and change the file. But I am unsure how that would be done in that package, and also unsure how that would change the output-file. I need to check this further.